apps/examples/testcase/le_tc/kernel/tc_libc_misc.c

```diff
@@ -168,6 +179,164 @@ static void tc_libc_misc_crc16part(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+ * @fn                  :tc_libc_misc_crc16_ccitt_xmodem
+ * @brief               :Return a 16-bit CRC16-CCITT/Xmodem of the contents of the 'src' buffer, length 'len'
+ * @scenario            :Return a 16-bit CRC16-CCITT/Xmodem of the contents of the 'src' buffer, length 'len'
+ * @API's covered       :crc16_ccitt_xmodem
+ * @Preconditions       :None
+ * @Postconditions      :None
+ * @Return              :void
+ */
+static void tc_libc_misc_crc16_ccitt_xmodem(void)
+{
+	uint16_t ret_chk;
+	uint8_t src_arr[ARR_NUM] = { VAL_50, VAL_100 };
+	uint16_t res_arr[ARR_NUM] = { VAL_5649, VAL_11298 };
+	size_t length = 1;
```

Comment:
This is unchanged. It looks we don't need to use a variable for this. How about using just 1 at line 201 directly?

Reply:
It doesn't matter, but I think it would be better to use length as #define for readablility. Is it okay?